usaspending_api/search/models/subaward_search.py

```diff
@@ -0,0 +1,450 @@
+
+    funding_agency = models.ForeignKey(
+        "references.Agency", models.DO_NOTHING, related_name="funding_subawardsearch", null=True
+    )
+    funding_toptier_agency_name = models.TextField(null=True, blank=True)
+    funding_toptier_agency_abbreviation = models.TextField(null=True, blank=True)
+    funding_subtier_agency_name = models.TextField(null=True, blank=True)
+    funding_subtier_agency_abbreviation = models.TextField(null=True, blank=True)
+
+    cfda = models.ForeignKey("references.Cfda", models.DO_NOTHING, related_name="related_subawardsearch", null=True)
+    cfda_number = models.TextField(null=True, blank=True)
+    cfda_title = models.TextField(null=True, blank=True)
+
+    # USAS Derived Fields
+    sub_fiscal_year = models.IntegerField()
+    sub_total_obl_bin = models.TextField()
+    sub_awardee_or_recipient_legal = models.TextField(null=True, blank=True)
+    sub_ultimate_parent_legal_enti = models.TextField(null=True, blank=True)
+    business_type_code = models.TextField(null=True, blank=True)
+    business_categories = ArrayField(models.TextField(), default=list)
+    treasury_account_identifiers = ArrayField(models.IntegerField(), default=None)
+    pulled_from = models.TextField(null=True, blank=True)
+    type_of_contract_pricing = models.TextField(null=True, blank=True)
+    type_set_aside = models.TextField(null=True, blank=True)
+    extent_competed = models.TextField(null=True, blank=True)
+    product_or_service_code = models.TextField(null=True, blank=True)
+    product_or_service_description = models.TextField(null=True, blank=True)
+
+    sub_legal_entity_country_code = models.TextField(null=True, blank=True)
+    sub_legal_entity_country_name = models.TextField(null=True, blank=True)
+    sub_legal_entity_county_code = models.TextField(null=True, blank=True)
+    sub_legal_entity_county_name = models.TextField(null=True, blank=True)
+    sub_legal_entity_zip5 = models.TextField(null=True, blank=True)
+    sub_legal_entity_city_code = models.TextField(null=True, blank=True)
+    sub_legal_entity_congressional = models.TextField(null=True, blank=True)
+
+    place_of_perform_scope = models.TextField(null=True, blank=True)
+    sub_place_of_perform_country_co = models.TextField(null=True, blank=True)
+    sub_place_of_perform_country_name = models.TextField(null=True, blank=True)
+    sub_place_of_perform_county_code = models.TextField(null=True, blank=True)
+    sub_place_of_perform_county_name = models.TextField(null=True, blank=True)
+    sub_place_of_perform_zip5 = models.TextField(null=True, blank=True)
+    sub_place_of_perform_city_code = models.TextField(null=True, blank=True)
+    sub_place_of_perform_congressio = models.TextField(null=True, blank=True)
+
+    # USAS Vectors
+    keyword_ts_vector = SearchVectorField()
+    award_ts_vector = SearchVectorField()
+    recipient_name_ts_vector = SearchVectorField()
+
+    class Meta:
+        db_table = 'rpt"."subaward_search'
```

Comment:
Same comment as before, and this is where it came from in the migration. Is this the right quoting syntax?

Reply:
(same response as the other)